tv/interfaces/index.ts

```diff
@@ -4,3 +4,22 @@ export interface Blog {
   blog_Quote: string;
```

Comment:
Could the blog interface be removed? It does not seem to be used anymore.

Reply:
Removed in speaker details branch